ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1357,3 +1357,23 @@ MAV_LANDED_STATE GCS_MAVLINK_Copter::landed_state() const
     }
     return MAV_LANDED_STATE_IN_AIR;
 }
+
+float GCS_MAVLINK_Copter::vfr_hud_airspeed() const
+{
+    // airspeed sensors are best.  While the AHRS airspeed_estimate
+    // will use an airspeed sensor, that value is constrained by the
+    // ground speed.  When reporting we should send the true airspeed
+    // value if possible:
+    if (copter.airspeed.enabled() && copter.airspeed.healthy()) {
+        return copter.airspeed.get_airspeed();
+    }
+
+    // airspeed estimates are OK:
```

Comment:
Should see if @tridge thinks we should pass the estimate through in Plane as well - and this move this code up.